### PR TITLE
Hotfix broken redirects

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Reused `react-router` navigation instead of `window.location` manipulations

### CODE REVIEW

**Review by kgevorkyan — APPROVED** [review]

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt [hunk 1]
```diff
@@ -74,7 +75,7 @@ class FallbackView : AbstractView<FallbackViewProps, State>(Style.SAVE_LIGHT) {
             }
 
             if (props.withRouterLink == true) {
-                Link {
+                Navigate {
                     to = "/"
```
Comment (kgevorkyan) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt:78]: may be create global `homePage = /` ?

**Review by kgevorkyan — APPROVED** [review]

**Review by Cheshiriks — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt:78]: Not sure what you mean. New `FC`?

Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt:78]: I guess he means
```kotlin
# Constants.kt
const val homePage = "/"
```

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt:78]: oh
not sure if it is needed

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt:78]: will merge now, later will fix if needed

**Pull request merged**